core/blockstm/executor.go

```diff
@@ -376,16 +376,6 @@ func (pe *ParallelExecutor) Prepare() error {
 		}(i)
 	}
 
-	pe.settleWg.Add(1)
```

Comment:
`settleWg` is now unused, the only remaining usage is a `settleWg.Wait()` that will do nothing.

Reply:
Thanks for taking a look! Will remove settleWg.